.github/workflows/release.yml

```
@@ -0,0 +1,341 @@
+        Architecture: amd64
+        Maintainer: Vibe Ensemble Team <hello@vibe-ensemble.dev>
+        Description: MCP server for coordinating multiple Claude Code instances
+         The Vibe Ensemble MCP server enables distributed task execution with 
+         unified management, communication, and issue tracking capabilities.
+        EOF
+        
+        # Build DEB package
+        dpkg-deb --build packaging/deb/vibe-ensemble-mcp
+        mv packaging/deb/vibe-ensemble-mcp.deb dist/vibe-ensemble-mcp_${{ needs.create-release.outputs.version }}_amd64.deb
+    
+    - name: Create RPM package  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)